packages/libs/react-ui/src/components/Link/Link.tsx

```diff
@@ -0,0 +1,37 @@
+import { SystemIcon } from '../Icons';
+
+import { iconContainerClass, linkContainerClass } from './Link.css';
+
+import React from 'react';
+
+export interface ILinkProps {
+  href: string;
+  target?: '_blank' | '_self' | '_parent' | '_top';
+  children: React.ReactNode;
+  iconPosition?: 'left' | 'right';
+  hasIcon?: boolean;
```

Comment:
We should follow the same subcomponent pattern used in the button. So instead of passing hasIcon and IconPosition, expose a Link.Icon component that the user can position